docs/rfc/rfc-1.adoc

```diff
@@ -361,22 +361,6 @@ If the depositor does not pay to unlock their account balance within the
 `skip_sweep_timeout`, then the system siezes the balance, mints TBTC, uses it
```

Comment:
It seems this document contains a bunch of typos.
Could you fix them as well?

critial - > critical
sortiton - > sortition
tranfered -> transferred
Addtionally -> Additionally
tranaction -> transaction (2 times)
occured -> occurred
accomodate -> accommodate
conract -> contract
siezes -> seizes

Reply:
Oh my... and I used Grammarly when writing it... 🤔

Reply:
But it was done during previous commits :)

Reply:
I see. Let's then address it in a separate PR to not increase the diff of this one.